### PR TITLE
New day2 step for openshift-qe perf metal

### PR DESCRIPTION
**Pull request by josecastillolema**

### CODE REVIEW

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.15-nightly-x86-node-density-heavy-6nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.15-nightly-x86-node-density-heavy-6nodes

File: ci-operator/step-registry/openshift-qe/installer/bm/day2/openshift-qe-installer-bm-day2-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  as: openshift-qe-installer-bm-day2
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  commands: openshift-qe-installer-bm-day2-commands.sh
+  credentials:
+  - namespace: test-credentials # this entry injects the custom credential
+    name: perfscale-metal-bastion
+    mount_path: /secret
```
Comment (jtaleric) [inline ci-operator/step-registry/openshift-qe/installer/bm/day2/openshift-qe-installer-bm-day2-ref.yaml:11]: You need to switch this to `/bm` as you have defined here:  https://github.com/openshift/release/pull/52001/files#diff-0aee60055ce70da0627bc5c3219de2c2a868fd1bef0523d3249492e1c4e9b10aR9 Or switch the command reference to use `/secret`

Comment (josecastillolema) [reply ci-operator/step-registry/openshift-qe/installer/bm/day2/openshift-qe-installer-bm-day2-ref.yaml]: Thanks!

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.15-nightly-x86-node-density-heavy-6nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.15-nightly-x86-node-density-heavy-6nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.15-nightly-x86-node-density-heavy-6nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.15-nightly-x86-node-density-heavy-6nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.15-nightly-x86-node-density-heavy-6nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.15-nightly-x86-node-density-heavy-6nodes

**Comment by josecastillolema** [comment]
@jtaleric looking good now, ptal

File: ci-operator/step-registry/openshift-qe/installer/bm/day2/openshift-qe-installer-bm-day2-ref.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    documentation: |-
+     Label to apply to worker nodes, if any
+  - name: LABEL_NUM_NODES
+    default: "1"
+    documentation: |-
+     Number of workers to apply the label
+  - name: SRIOV
+    default: "true"
+    documentation: |-
+     Wether to enable or not SRIOV
```
Comment (jtaleric) [inline ci-operator/step-registry/openshift-qe/installer/bm/day2/openshift-qe-installer-bm-day2-ref.yaml:32]: enable SRIOV

**Review by jtaleric — APPROVED** [review]
overall lgtm simple doc wording nit.

File: ci-operator/step-registry/openshift-qe/installer/bm/day2/openshift-qe-installer-bm-day2-ref.yaml (hunk 2 above)
Comment (josecastillolema) [reply ci-operator/step-registry/openshift-qe/installer/bm/day2/openshift-qe-installer-bm-day2-ref.yaml]: Done

**Pull request merged**